ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -397,11 +397,11 @@ tests:
   interval: 24h
   steps:
     cluster_profile: openstack-nfv
+    dependencies:
+      OO_INDEX: performance-addon-operator-index-gen
```

Comment:
This will not work in the context of an openshift/release job. This relies on the index built by ci-operator, which can only be built in a job that belongs to (and has access to) the repository where the operator and bundle gets built.

Reply:
thanks. I'll do with another method then.